### ISSUE TITLE
Example request: Twitter Signin Integration

### ISSUE THREAD

**Issue by nisrulz** [nisrulz/flutter-examples#21]
Project Name: twitter_signin
Description: A flutter example app to Twitter Signin

**Comment by sanyamdogra** [comment]
Can i work on this ?

**Comment by nisrulz** [comment]
Yes, please. Make sure you follow the guidelines specified in the Readme. Take other examples as reference.

**Comment by hiashutoshsingh** [comment]
@nisrulz Hey, is this is not taken i can take this issue.

**Comment by nisrulz** [comment]
The idea is to have firebase signin support. That way we can enable other signin options too. So twitter is supposed to be the first version.

If you created multiple files within the same app project regarding signing code for each type of possible sign provider and then later using a button allow to signin with each then the example app would be complete.

Firebase allows that to be done pretty easily [Read more](https://firebase.flutter.dev/docs/auth/social)

If you did that the example would be named as `using_firebase_auth`

If you go the route to setup only twitter auth, then name your example app to be `twitter_signin`. Once this is done, there would be more apps like that such as `google_signin`, `apple_signin`, etc.

I am in favor of 1 single app called `using_firebase_auth`, with code for each signing provider in different files so it is still easier for beginners to grasp.

**Comment by Rocktopus101** [comment]
Hi! I would like to work on this. Can you assign this to me?

**Comment by Sriraj-dev** [comment]
@nisrulz Hey, Can I work on this feature of using_firebase_auth which includes the firebase authentication and Google_Signin using firebase.

**Comment by nisrulz** [comment]
> @nisrulz Hey, Can I work on this feature of using_firebase_auth which includes the firebase authentication and Google_Signin using firebase.

This issue is about Twitter Sign in. If you want to open a PR please do so with Firebase Auth  + Twitter Signin

**Comment by Sriraj-dev** [comment]
> > @nisrulz Hey, Can I work on this feature of using_firebase_auth which includes the firebase authentication and Google_Signin using firebase.
> 
> This issue is about Twitter Sign in. If you want to open a PR please do so with Firebase Auth + Twitter Signin

Okay , I will work on this if this is not assigned yet.

**Comment by razaAtif** [comment]
Hi @nisrulz ,
My name is Atif Raza, I am a flutter developer and  a beginner in open source, I would like to work on this issue , can you please assign this issue to me.

**Comment by upahar-khatiwada** [comment]
hello i want to complete this please
i will do it within today

**Comment by upahar-khatiwada** [comment]
@nisrulz can you possibly review this?
